queries.sql

```diff
@@ -39,4 +39,67 @@ select count(animal_escape_attempts) from animal group by neutered;
 select min(animal_weight_kg) from animal;
 select max(animal_weight_kg) from animal;
 SELECT species, ROUND(AVG(animal_escape_attempts)) FROM animal
-  WHERE animal_date_of_birth >= '1990-01-01' AND animal_date_of_birth <= '2000-12-31' GROUP BY species;
\ No newline at end of file
+  WHERE animal_date_of_birth >= '1990-01-01' AND animal_date_of_birth <= '2000-12-31' GROUP BY species;
+
+-- What animals belong to Melody Pond?
+select animal.owner_id, animal.animalname, owners.fullname, owners.id from animal join owners on animal.owner_id = owners.id where owners.fullname = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+select animal.species_id, animal.animalname, species.id, species.fullname from animal join species on animal.species_id = species.id where species.fullname = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+select animal.owner_id, animal.animalname, owners.fullname, owners.id from animal join owners on animal.owner_id = owners.id where animal.owner_id = null;
+
+-- How many animals are there per species?
+select count(animal.species_id), species.id, species.fullname from animal join species on animal.species_id = species.id group by species.id;
+
+-- List all Digimon owned by Jennifer Orwell.
+select species.id, owners.id, owners.fullname from animal join species on animal.species_id  =species.id join owners on animal.owner_id = owners.id where owners.fullname = 'Jennifer Orwell';
+
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+select animal.animal_escape_attempts, owners.fullname from animal join owners on animal.owner_id = owners.id   WHERE animal.animal_escape_attempts=0
+AND owners.fullname = 'Dean Winchester';
+
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+select owners.fullname, count(*)  from animal join owners on animal.owner_id = owners.id  group by owners.fullname;
+
+
+-- Who was the last animal seen by William Tatcher?
+select visits.animal_id, visits.vets_id from visits where visits.vets_id =1 order by visits.animal_id desc limit 1;
+
+-- How many different animals did Stephanie Mendez see?
+select count(visits.animal_id) from visits where visits.vets_id = 3;
+
+-- List all vets and their specialties, including vets with no specialties.
+select vets.name, specializations.vets_id from vets left join specializations on vets.id = specializations.vets_id;
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+select * from visits where visits.vets_id =3 and visits.date_visit between '2020-04-01' and '2020-08-30';
```

Comment:
- [ ] Good job working on these queries but please kindly utilize join which is the main essence of this milestone.
- [ ] Also, please include a query which returns the right result for `What animal has the most visits to vets?`